src/context/XrplContext.ts

```diff
@@ -23,12 +35,16 @@ class XrplContext {
     public xrplAcc: any;
     public multiSigner: MultiSigner;
     public voteContext: VoteContext;
+    public network: string;
 
     public constructor(hpContext: HotPocketContext, address: string, secret: string | null = null, options: XrplOptions = {}) {
         this.hpContext = hpContext;
         this.voteContext = hpContext.voteContext;
+        this.network = options?.network ? options.network : "mainnet";
+        const rippleServer = options?.rippleServer ? options.rippleServer : NETWORKS[this.network].rippledServer;
+
         // autoReconnect: false - Do not handle connection failures in XrplApi to avoid contract hanging.
-        this.xrplApi = options.xrplApi || new evernode.XrplApi(null, { autoReconnect: false });
+        this.xrplApi = options.xrplApi || new evernode.XrplApi(rippleServer, { autoReconnect: false, fallbackRippledServers: options?.fallbackRippledServers });
```

Comment:
Shall we move this xrplAPi, xrplAcc and multiSigner part to init method so we don't have to hardcode `NETWORKS` constant in the code